source/material_model/viscoelastic.cc

```diff
@@ -144,7 +144,19 @@ namespace aspect
                                               const double average_elastic_shear_modulus,
                                               const double dte) const
     {
```

Comment:
@naliboff -- can I ask you to take a look at whether you think this change here is in this file is reasonable? I want to make a change where the time step is a signaling_NaN before we start the simulation (it used to be zero) because it really has not been computed or set at that time, and this now trips up the code here.